src/helics/shared_api_library/MessageFederate.h

```diff
@@ -489,7 +489,7 @@ HELICS_EXPORT int helicsMessageGetMessageID(helics_message_object message);
  *
  * @return The flags associated with a message.
  */
-HELICS_EXPORT helics_bool helicsMessageCheckFlag(helics_message_object message, int flag);
+HELICS_EXPORT helics_bool helicsMessageCheckFlag(helics_message_object message, helics_federate_flags flag);
```

Comment:
I'm pretty sure this shouldn't be `helics_federate_flags` but not sure what it should be.

Reply:
The message has a 16 bit flag field some of which are purposely left user defined, so there is a set which HELICS uses but others which it doesn't, so if we used an enum it would need to define 16 enumerations from 0-15